proxy/src/coco.rs

```diff
@@ -230,7 +223,7 @@ pub fn local_branches(repo_path: &str) -> Result<Vec<Branch>, error::Error> {
     let mut branches = browser
         .list_branches(None)?
         .into_iter()
-        .map(|b| Branch(b.name.name()))
+        .map(|b| Branch(b.name.name().to_string()))
```

Comment:
Note to self: this is just bad... I'll make an issue to have a `.name()` function for `Branch`

Reply:
@FintanH Should I create an issue for it?

Reply:
I already did :) radicle-dev/radicle-surf#68